src/languageSupport/languages/flux/lsp/utils.ts

```diff
@@ -58,4 +110,12 @@ export enum Methods {
   Hover = 'textDocument/hover',
   References = 'textDocument/references',
   Rename = 'textDocument/rename',
+  ExecuteCommand = 'workspace/executeCommand',
+}
+
+export enum ExecuteCommand {
+  InjectionMeasurement = 'injectMeasurementFilter',
+  InjectField = 'injectFieldFilter',
+  InjectTag = 'injectTagFilter',
+  InjectTagValue = 'injectTagValueFilter',
```

Comment:
I'm glad we have these. I _think_ there might be a way to simplify this serialization, which also might mean these get renamed. I would not consider this API "stable" yet, so this is good that we have these all in the same place.